src/test/java/TestDebugEntry.java

```diff
@@ -35,47 +34,6 @@ public void cleanupTests() {
     subsystem2 = new TestSubsystem();
   }
 
-  @Test()
-  public void raiseDupEntryError() {
-    // Duplicate Entry
-    if (Robot.isCompetition) {
-      return;
-    }
-    try (MockedStatic<DriverStation> mockedFactory = Mockito.mockStatic(DriverStation.class)) {
-      mockedFactory
-          .when(() -> DriverStation.reportError(anyString(), anyBoolean()))
-          .thenCallRealMethod();
-      new DebugEntry<Double>(0.0, "test", subsystem);
-      new DebugEntry<Double>(0.0, "test", subsystem);
-      mockedFactory.verify(() -> DriverStation.reportError(anyString(), anyBoolean()), times(1));
-    }
-  }
-
-  @Test
```

Comment:
instead of deleting tests, you can add "@Disabled" above them so they will be skipped. We can revisit them another time to see if they are still relevant to the new DebugEntry